### PR TITLE
adding devtools tasks to command palette

### PR DESCRIPTION
**Pull request by alex-gable**
# What problem did you solve?
while the majority of `devtools::*` package develop/build shortcuts were available, `devtools::check()` was missing.

## (If you have)Screenshot

R: Check available in Command Palette and sends to R console

## (If you do not have screenshot) How can I check this pull request?
Build version > `Cmd + Shift + P` > Type `R: Check` > Press Enter

### CODE REVIEW

**Comment by ElianHugh** [comment]
I wonder if it would be worth moving away from the runTextInTerm paradigm for these commands (I.e. commands that don't influence/affect the R environment). Is there a benefit to running in the terminal as opposed to a task? We have the devtools::check task definition but iirc no command palette bbinding

**Comment by renkun-ken** [comment]
I think it should be a task instead of sending code to terminal.

**Comment by alex-gable** [comment]
I believe that's how RStudio treats it as well, for whatever that's worth. Should that be in the scope of this PR though?

**Comment by ElianHugh** [comment]
I think for this PR, it would probably be worth pointing the command palette to the task binding. We can do the same for the other command palette bindings in a future PR

**Comment by alex-gable** [comment]
Understood. Just to be transparent, that's likely outside of my skill set

**Comment by ElianHugh** [comment]
No worries, you're 90% there already! I think you would only need to change the command and its arguments. E.g., in my keybindings I have the following:

```
{		
  "key": "ctrl+shift+t",		
  "command": "workbench.action.tasks.runTask",		
  "args": "R: Test"
},
```

**Comment by alex-gable** [comment]
okay, I _think_ this is the solution you alluded to, @ElianHugh, I appreciate the pointer. I did a bit of research and there's definitely more complex (and I think appropriate) ways to accomplish this. see: microsoft/vscode#40758.

any how. I took a couple of liberties in terms of naming and organization. I wanted to be sure the commands that match `Check`, which execute outside of the Terminal, all had tasks. so I added a task for `Build` and then grouped these task commands separately from the other editor independent commands. 

I then categorized these "Package" command contributions separately in the because, as I was testing, it made the command in the palette clearer.

Feel free to revert/change whatever!

File: package.json [hunk 1]
```diff
@@ -422,28 +422,33 @@
...
+        "category": "R Package",
         "command": "r.loadAll"
       },
       {
-        "title": "Test Package",
-        "category": "R",
-        "command": "r.test"
+        "title": "Build",
+        "category": "R Package",
+        "command": "r.build"
```
Comment (ElianHugh) [inline package.json:431]: This should probably point to r.buildTask (currently errors out)

**Comment by ElianHugh** [comment]
Thank you for the nice work you've done! The commands work nicely (barring the one issue I've noted above), and I think the naming is appropriate too. I don't have a strong opinion re: naming the commands r.commandTask, but it may cause unexpected  issues for VSCode-R users that have shortcuts pointing to the old command names (e.g. r.test vs r.testTask)

**Review by ElianHugh — APPROVED** [review]
LGTM

**Comment by ElianHugh** [comment]
Anything else to add? @renkun-ken

**Comment by renkun-ken** [comment]
LGTM sorry for late reply!

**Pull request merged**